plugins/inputs/ipset/ipset_entries_test.go

```diff
@@ -0,0 +1,96 @@
+package ipset
+
+import (
+	"testing"
+	"time"
+
+	"github.com/influxdata/telegraf"
+	"github.com/influxdata/telegraf/testutil"
+	"github.com/stretchr/testify/require"
+)
+
+func TestIpsetEntries(t *testing.T) {
+	acc := new(testutil.Accumulator)
+
+	lines := []string{
+		"create mylist hash:net family inet hashsize 16384 maxelem 131072 timeout 300 bucketsize 12 initval 0x4effa9ad",
+		"add mylist 89.101.238.143 timeout 161558",
+		"add mylist 122.224.15.166 timeout 186758",
+		"add mylist 47.128.40.145 timeout 431559",
+	}
+
+	entries := ipsetEntries{}
+	for _, line := range lines {
+		require.NoError(t, entries.addLine(line, acc))
+	}
+	entries.commit(acc)
```

Comment:
As the accumulator is no pointer anymore...
```suggestion
		require.NoError(t, entries.addLine(line, &acc))
	}
	entries.commit(&acc)
```